polaris-tokens/src/token-groups/colors.ts

```diff
@@ -576,6 +576,75 @@ export const colors = {
   'surface-attention': {
     value: 'rgba(255, 234, 138, 1)',
   },
+  'surface-magic': {
+    value: 'rgba(233, 217, 252, 1)',
+    description: 'For use as background color of Shopify Magic components.',
+  },
+  'surface-magic-subdued': {
+    value: 'rgba(246, 240, 253, 1)',
+    description:
+      'For use as subdued background color of Shopify Magic components.',
+  },
+  'surface-magic-subdued-hovered': {
+    value: 'rgba(238, 223, 255, 1)',
+    description: 'Hover state for subdued Shopify Magic surfaces.',
+  },
```

Comment:
The naming of these tokens makes sense internally. Exposing these to developers would be confusing. Is there any context or names that would be better suited so that our developer/designer community understand when to use them?

Reply:
Good question. If 3rd party folks will have access to build on top of it then keeping the name probably makes most sense and in time this concern goes away. If this is a internal thing only then idk what is a better option because updating the name for 3p folks likely confuses 1p ones since internally this is called Magic.